cli/flox/src/commands/services/logs.rs

```diff
@@ -37,7 +37,7 @@ impl Logs {
         subcommand_metric!("services::logs");
 
         let env = ServicesEnvironment::from_environment_selection(&flox, &self.environment)?;
-        guard_service_commands_available(&env)?;
+        guard_service_commands_available(&env, &flox.system)?;
```

Comment:
suggestion nonblocking: if it takes a lot of effort I'm not sure if it's worth it, but we could make the errors for all subcommands consistent with the error you added for start.
```
flox [main] % flox list -c
version = 1
[services]
one.command = "sleep infinity"
invalid.command = "sleep infinity"
invalid.systems = ["dummy-system"]
flox [main] % flox services stop invalid
❌ ERROR: Service 'invalid' not found.
```

Reply:
Oh also incidentally `start` is inconsistent depending on whether you have process-compose running:
```
flox [main] % flox services start invalid
❌ ERROR: Service 'invalid' not found.

flox [main] % flox services stop
✅ Service 'one' stopped

flox [main] % flox services start invalid
❌ ERROR: Service 'invalid' not available on 'aarch64-darwin'.
```